deliver/lib/deliver/app_screenshot.rb

```diff
@@ -143,6 +151,9 @@ def self.device_messages
 
     def self.devices
       return {
+        ScreenSize::IOS_58 => [
+          [1125, 2436]
```

Comment:
@cherpake 

Thank you for reviewing 👍 
Do you mean we need to add `[2436, 1125]`?
https://github.com/fastlane/fastlane/pull/10731#issuecomment-340258314

I think this is enough because we handle orientation in `def calculate_screen_size`

Reply:
Thank you for your work. It works.

Reply:
Why is the iPhone X resolution called "iphone58" (display size) whereas all previous screenshot sizes are called like the device? (iphone6Plus and not iphone55)

Reply:
Hi @max-ott 👋 

Apple use `iphone35` for 3.5 inch devices (iPhone 3G - iPhone 4S) and `iphone4` for 4.0 inch devices (iPhone 5 - iPhone 5s).
So `iphone58` is not exception :)